packages/api-db-mongodb/src/migration.ts

```diff
@@ -593,6 +593,26 @@ export const Migrations: Migration[] = [
         }
       )
     }
+  },
+  {
+    //  Rename street field in address to address
+    version: 14,
+    async migrate(db, locale) {
+      const users = await db.collection(CollectionName.Users)
+      await users.updateMany(
+        {
+          emailVerified: {$exists: false}
+        },
+        {$set: {emailVerified: null}}
+      )
+
+      await users.updateMany(
+        {
+          oauth2Accounts: {$exists: false}
+        },
+        {$set: {oauth2Accounts: []}}
```

Comment:
can't you add this attr to the upper mutation? why two?

Reply:
because it's should only set the field it it doesn't exist.